### PR TITLE
fix: [PL-38333]: Change severity for non-critical Error messages from delegate

### PR DESCRIPTION
**Pull request by raghuAtWings**
## Describe your changes
Changing log levels to info for a few very high non actionable error logs. This is for the effort of reducing error logs 

**Tests done**
No test done since this is only log level change. 

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger utAll`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/47099)

### CODE REVIEW

File: 960-api-services/src/main/java/io/harness/artifactory/ArtifactoryClientImpl.java [hunk 1]
```diff
@@ -339,10 +339,10 @@ public Map<String, String> getRepositories(
         log.debug("Retrieving repositories for packages {} success", packageTypes.toArray());
       }
     } catch (SocketTimeoutException e) {
-      log.error(ERROR_OCCURRED_WHILE_RETRIEVING_REPOSITORIES, e);
+      log.info(ERROR_OCCURRED_WHILE_RETRIEVING_REPOSITORIES, e);
       return repositories;
     } catch (Exception e) {
-      log.error(ERROR_OCCURRED_WHILE_RETRIEVING_REPOSITORIES, e);
+      log.info(ERROR_OCCURRED_WHILE_RETRIEVING_REPOSITORIES, e);
```
Comment (gn-harness) [inline 960-api-services/src/main/java/io/harness/artifactory/ArtifactoryClientImpl.java:345]: This one we are rethrowing - do we need to log it here?

File: 930-delegate-tasks/src/main/java/io/harness/delegate/task/common/AbstractDelegateRunnableTask.java [hunk 2]
```diff
@@ -147,8 +147,7 @@ private void runDelegateTask() {
       taskResponse.responseCode(ResponseCode.RETRY_ON_OTHER_DELEGATE);
     } catch (Throwable throwable) {
       if (!(throwable instanceof Exception) || !isSupportingErrorFramework()) {
-        log.error(
-            format("Unexpected error while executing delegate taskId: [%s] in accountId: [%s]", taskId, accountId),
+        log.info(format("Unexpected error while executing delegate taskId: [%s] in accountId: [%s]", taskId, accountId),
```
Comment (gn-harness) [inline 930-delegate-tasks/src/main/java/io/harness/delegate/task/common/AbstractDelegateRunnableTask.java:150]: this should be warning?

File: 930-delegate-tasks/src/main/java/io/harness/delegate/task/citasks/cik8handler/CIK8CleanupTaskHandler.java [hunk 3]
```diff
@@ -193,11 +193,11 @@ private boolean deleteSecrets(
             isDeletedContainerImageSecret =
                 cik8JavaClientHandler.deleteSecret(coreV1Api, namespace, containerSecretName);
           } catch (Exception ex) {
-            log.error("Failed to delete secret {}", secretName, ex.getMessage());
+            log.info("Failed to delete secret {}", secretName, ex.getMessage());
```
Comment (gn-harness) [inline 930-delegate-tasks/src/main/java/io/harness/delegate/task/citasks/cik8handler/CIK8CleanupTaskHandler.java:196]: warning?

**Review by gn-harness — APPROVED** [review]

File: 960-api-services/src/main/java/io/harness/artifactory/ArtifactoryClientImpl.java (hunk 1 above)
Comment (raghuAtWings) [reply 960-api-services/src/main/java/io/harness/artifactory/ArtifactoryClientImpl.java:345]: not sure about this

**Review by gn-harness — DISMISSED** [review]

**Comment by raghuAtWings** [comment]
trigger uts

**Comment by raghuAtWings** [comment]
trigger utAll

**Comment by raghuAtWings** [comment]
trigger utAll

**Comment by raghuAtWings** [comment]
trigger uts

**Review by gn-harness — APPROVED** [review]

**Pull request merged**